src/Apps/Order/Components/AddressForm.tsx

```diff
@@ -203,7 +203,7 @@ export class AddressForm extends React.Component<
             />
           </Flex>
         </TwoColumnSplit>
-        {(!this.props.billing || this.props.showPhoneNumberInput) && (
+        {this.props.showPhoneNumberInput && (
```

Comment:
Hey @dblandin you added this check right? Is it ok if I remove it?

Reply:
@lilyfromseattle I think this might be still used. We show this form in user profile payment methods as well.

Reply:
From what I can see it's used in BidForm, RegistrationForm, PaymentPicker, and PaymentForm. In all places `billing` is true, so removing the `!this.props.billing` check shouldn't affect the other usages.

Reply:
Yeah, I'm having trouble finding an example of when `billing` is et to `false` as well. One thing @sweir27 and I had in mind when consolidating the payment forms was not showing "Required for shipping logistics" on those other forms. So feel free to remove this condition but I would check those other places to make sure everything looks okay.

Reply:
could it be add new payment method under user's setting or during bidding registration?